### PR TITLE
Gives yogsdelta chemistry a double door

### PR DESCRIPTION
**Pull request by Takahiru**
## Intent of your Pull Request

It's annoying without it
#### Changelog

:cl:  
tweak: gives yogsdelta chemistry a double windoor
/:cl:

### CODE REVIEW

**Review by patpol4 — CHANGES_REQUESTED** [review]
1. Update PR with the other change you did
2. Inner windoor access should be just 33
3. Outer windoor does not need an access var, The default is the same as 0 
4. Change changelog to say windoor instead of door

**Comment by Takahiru** [comment]
1. No
2. I have it at the old access???
3. blame fastdmm2
4. you are nitpicking and biased

**Comment by patpol4** [comment]
1. Turns out there is no conflict so it should be fine
2. Ok it should stay on 5,33 , delta has low iq access restrictions
3. Check the other chemistry windoor, it has no var. In fastdmm2 , just go to the variable, delete the value, press enter, it wont save it that way but still keep it at no access needed. This is a way of preventing unneeded variable bloat.
4. Thanks

**Comment by Takahiru** [comment]
3. :thonk: it dosen't appear to be going away

**Review by patpol4 — APPROVED** [review]
It went away, you can check it by looking at if the variable is white or not, 
chrome_SB3KpKc0sZ
chrome_mknQniGAXS

File: _maps/map_files/YogsDelta/YogsDelta.dmm [hunk 1]
```diff
@@ -127528,6 +127510,27 @@
...
 /area/engine/supermatter)
+"vjX" = (
+/obj/structure/table/reinforced,
+/obj/machinery/door/poddoor/shutters/preopen{
+	id = "chemisttop";
+	name = "Chemistry Lobby Shutters"
+	},
+/obj/item/folder/yellow,
+/obj/machinery/door/window/northleft{
+	name = "Chemistry Desk"
```
Comment (patpol4) [inline _maps/map_files/YogsDelta/YogsDelta.dmm:127521]: And it's not in here

**Pull request merged**